mlos_bench/mlos_bench/config/environments/apps/fio/fio.jsonc

```diff
@@ -0,0 +1,70 @@
+{
+    "name": "FIO Composite Env",
+    "class": "mlos_bench.environments.composite_env.CompositeEnv",
+
+    "config": {
+        "children": [
+            {
+                "name": "Upload FIO job file",
+                "class": "mlos_bench.environments.local.LocalFileShareEnv",
+                "config": {
+                    "required_args": [
+                        "experiment_id",
+                        "trial_id"
+                    ],
+                    "upload": [
+                        {
+                            "from": "environments/apps/fio/scripts/remote/",
+                            "to": "$experiment_id/$trial_id/scripts"
+                        }
+                    ]
+                }
+            },
+            {
+                "name": "Run FIO remotely",
+                "class": "mlos_bench.environments.remote.RemoteEnv",
+                "config": {
+                    "required_args": [
+                        "vmName",
+                        "experiment_id",
+                        "trial_id",
+                        "mountPoint"
+                    ],
+                    "setup": [
+                        "sudo apt-get update",
+                        "sudo apt-get -y install fio"
```

Comment:
This is fine for now, but one thing about needing to support Mariner in the future is we're going to have to special case which package manager to use in a lot of cases and that will probably mean converting these to more complicated scripts we download/run.

Reply:
yep. this config is open source, so I think `apt-get` should be fine for the sample config